docs/INSTALL.md

```diff
@@ -0,0 +1,214 @@
+
+5. **VM Control**: Clients can start, stop, and restart their VMs.
+
+6. **Configuration Modification**: Clients can modify server configurations to suit their needs.
+
+7. **Bandwidth Usage Graph**: Clients can view bandwidth usage graphs for intervals spanning 24 hours, a week, a month, and a year.
+
+## Installation
+
+### Downloading and Uploading to WHMCS
+
+1. Obtain the module from our GitHub repository.
+2. Unpack the module on your local system and transfer the "upCloudVps" folder to your WHMCS directory named "/path-to-your-WHMCS/modules/servers/" on your WHMCS installation.
+
+### Obtaining API Credentials
+
+1. Access the URL "https://hub.upcloud.com/dashboard" and log in using your credentials.
+2. Navigate to the "People" section from the left sidebar or directly visit "https://hub.upcloud.com/people/accounts".
+3. Proceed to create a sub-account by selecting the "Create subaccount" button or visit "https://hub.upcloud.com/people/accounts/new".
+4. Enter a username, password, and complete other required details (Personal Information), then click "Create subaccount" at the bottom.
+5. After creating the user, click on "Edit", then navigate to the Permissions section and click on "Go to permissions".
+6. Enable all permissions except "Control Panel" and save the changes.
+
+
+
+### Creating Your First WHMCS Server
+
+Reference: https://docs.whmcs.com/Servers
+
+1. Log in to your WHMCS as an admin user and go to Configuration > System Settings > Servers.
+2. Select "Add New Server" and choose "UpCloud VPS" from the module dropdown.
+3. Enter the Hostname or IP Address as: api.upcloud.com (you can assign any name).
+4. For Username, input your API username created in the previous step.
+5. For Password, input your API password created in the previous step.
+6. Proceed by clicking on "Test Connection" or "Continue Anyway".
+
+### Creating a Server Group
+
+1. Click Create New Group under the Options heading.
+2. Enter a name for your group.
+3. Select the servers you want to assign (previously created) to this group in the box on the left.
+4. Click Add to move them to the box on the right, which contains the servers for this group.
+5. Click Save Changes to complete the process.
+
+## Configuration
+
+### Creating a Product Group
+
+1. Navigate to Configuration > System Settings > Product/Services
+
+![Producs & Services](images/products_services.png)
+
+
+2. Click on Create Product Group
+
+![Product Group](images/product_group.png)
+
+
+3. Provide a Product Group name and adjust all other settings based on your requirements
+
+![Product Config](images/product_config.png)
+
+4. Click on Save Changes
+
+
+### Creating a New Product
+
+1. Click on Create a New Product
+
+![New Product](images/new_product.png)
+2. Configure the following details:
+   - **Product Type**: Select as "Other" (recommended to avoid prompting customers for Hostname, Password, NS1 & NS2 prefix during order process)
```

Comment:
This has some other positive (~required) consequences, not sure if enumerating them here is necessary

```suggestion
   - **Product Type**: Select as "Other"
```